tests/test_PsfexPsf.py

```diff
@@ -28,6 +28,7 @@
 import lsst.afw.image as afwImage
 import lsst.afw.detection as afwDetection
 import lsst.afw.geom as afwGeom
+import lsst.geom as geom
```

Comment:
You couldn't also remove the `afwGeom` above?

Reply:
No, because `makeSkyWcs` is used here.